apps/dotcms-ui/src/app/view/components/dot-image-editor/components/edit-controls/edit-controls.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+// Services
+import { StoreImageService } from '../../store-image.service';
+
+// Models
+import { Params } from '../../model/image-editor.model';
+
+@Component({
+  selector: 'dot-edit-controls',
+  templateUrl: './edit-controls.component.html',
+  styleUrls: ['./edit-controls.component.scss']
+})
+export class EditControlsComponent implements OnInit {
```

Comment:
All the classes in the all are prefixed with `Dot`

```suggestion
export class DotEditControlsComponent implements OnInit {
```